next.config.js

```diff
@@ -8,15 +13,8 @@ const nextConfig = {
       },
     ],
   },
-  // TODO: This has been merged into Next.js canary, but not in v13.4.9 yet
-  // https://github.com/vercel/next.js/pull/53902
-  modularizeImports: {
-    '@heroicons/react/24/solid': {
-      transform: '@heroicons/react/24/solid/{{member}}',
-    },
-    '@heroicons/react/24/outline': {
-      transform: '@heroicons/react/24/outline/{{member}}',
-    },
+  env: {
+    DEPLOY_PRIME_URL: process.env.DEPLOY_PRIME_URL,
```

Comment:
Brings this variable to the client. I suppose we could do `window.location` in most cases but this seemed like a good idea for the robots.txt file.